java-frontend/src/main/java/org/sonar/java/se/SymbolicExecutionVisitor.java

```diff
@@ -66,6 +66,7 @@ public MethodBehavior execute(MethodTree methodTree) {
         behaviorCache.put(methodTree.symbol(), methodBehavior);
         ExplodedGraphWalker walker = egwFactory.createWalker(this);
         methodBehavior = walker.visitMethod(methodTree, methodBehavior);
+        methodBehavior.completed();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) NullPointerException might be thrown as 'methodBehavior' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2259)